app/models/order.rb

```diff
@@ -13,6 +16,12 @@ class Order < ApplicationRecord
     ]
   end
 
+  module DetailType
+    SHIPMENT = 'Shipment'
```

Comment:
This is starting to feel like it should be a taxonomy table now? cc @patrixr  for thoughts